src/SonarScanner.MSBuild.Common/Telemetry/TelemetryUtils.cs

```diff
@@ -30,6 +30,28 @@ public static void AddTelemetry(ILogger logger, AggregatePropertiesProvider aggr
         }
     }
 
+    public static void AddTelemetry(ILogger logger, HostInfo serverInfo)
+    {
+        if (serverInfo is null)
+        {
+            return;
+        }
+
+        string serverUrl;
+        if (serverInfo.IsSonarCloud && serverInfo as CloudHostInfo is { } cloudServerInfo)
+        {
+            logger.AddTelemetryMessage("dotnetenterprise.s4net.serverInfo.region", cloudServerInfo.Region is "us" ? "us" : "default");
```

Comment:
We only support "us" as a valid value anyway. I don't think it is a problem to just use the region directly. I don't see how this could be a problem. This is not like a server name.
```suggestion
            logger.AddTelemetryMessage("dotnetenterprise.s4net.serverInfo.region", string.IsNullOrWhitespace(cloudServerInfo.Region) ? "default" : cloudServerInfo.Region;
```